### PR TITLE
UI and UX adjustments in new controllers.

### PR DESCRIPTION
**Pull request by or-else**

### CODE REVIEW

File: Tinodios/CredentialsChangeViewController.swift [hunk 1]
```diff
@@ -120,21 +118,6 @@ class CredentialsChangeViewController: UITableViewController {
...
     }
 
-    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        guard UIApplication.shared.applicationState == .active else {
-            return
-        }
-        self.setInterfaceColors()
-    }
-
-    private func setInterfaceColors() {
```
Comment (aforge) [inline Tinodios/CredentialsChangeViewController.swift:130]: Is it something we don't need anymore?

File: Tinodios/CredentialsChangeViewController.swift [hunk 2]
```diff
@@ -171,7 +154,11 @@ class CredentialsChangeViewController: UITableViewController {
             .then(onSuccess: { msg in
                 if let ctrl = msg?.ctrl, 200 <= ctrl.code && ctrl.code < 300 {
                     self.confirmationSectionVisible = true
-                    DispatchQueue.main.async { UiUtils.showToast(message: "Confirmation code sent", level: .info) }
+                    DispatchQueue.main.async {
+                        self.newTelField.isEnabled = false
```
Comment (aforge) [inline Tinodios/CredentialsChangeViewController.swift:158]: Nice!
Perhaps disable interaction with the entire view:
`self.view.isUserInteractionEnabled = false`

File: Tinodios/SettingsPersonalViewController.swift [hunk 3]
```diff
@@ -34,6 +34,8 @@ class SettingsPersonalViewController: UITableViewController {
     weak var me: DefaultMeTopic!
     private var imagePicker: ImagePicker!
 
+    private let phoneNumberKit = PhoneNumberKit()
```
Comment (aforge) [inline Tinodios/SettingsPersonalViewController.swift:37]: You can use `UiUtils.phoneNumberKit` to avoid creating another object.

**Review by aforge — APPROVED** [review]

File: Tinodios/CredentialsChangeViewController.swift (hunk 1 above)
Comment (or-else) [reply Tinodios/CredentialsChangeViewController.swift:130]: I'm using system colors. They are supposed to change automatically between dark and light themes.

File: Tinodios/CredentialsChangeViewController.swift (hunk 2 above)
Comment (or-else) [reply Tinodios/CredentialsChangeViewController.swift:158]: I want to disable just this field, not with the entire view. Am I missing something?

File: Tinodios/SettingsPersonalViewController.swift (hunk 3 above)
Comment (or-else) [reply Tinodios/SettingsPersonalViewController.swift]: Fixed

**Pull request merged**